check-sample.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+pids=""
+cd build
+./kvsWebrtcClientMaster SampleChannel &
+pids+=" $!"
+
+sleep 2
+./kvsWebrtcClientViewer SampleChannel &
+pids+=" $!"
+
+bash -c "sleep 10 && kill -s INT $(jobs -p)" &
```

Comment:
This is a good start. Let's make sure we add some configuration to the samples to run for a specified amount of time and exit gracefully. We also need to ensure that the samples actually communicate and exchange data. We can enhance that together with the samples refactor work.